### PR TITLE
Button

### PR DESCRIPTION
**Pull request by chanchal-panpaliya**
project live - https://solutionground.netlify.app/

### CODE REVIEW

**Review by Mohasin-Haque — COMMENTED** [review]
Code is good but again you need to work on sanity .. So please format the code , remove the unused code and use relative units.

File: button/Button/Button.css [hunk 1]
```diff
@@ -0,0 +1,262 @@
+@import "../../utility/component.css";
+
+.button{
+    width: 109px;
+    height: 40px;
+    text-align: center;
+    text-decoration: none;
+    border:none;
+    background-color: transparent;
+ }
```
Comment (haarsh24) [inline button/Button/Button.css:11]: Declare this color also and use.

File: button/Button/Button.css [hunk 2]
```diff
@@ -0,0 +1,262 @@
...
+   color: var(--white);
+   background-color: var(--icon-primary-color);
+   width: auto;
+   height: auto;
+ }
+
+/*------------*/ 
+
+.button-primary{
+    color: var(--white);
```
Comment (haarsh24) [inline button/Button/Button.css:95]: Give a proper name to this instead of white.

File: button/index.css [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+
+:root {
+  --icon-primary-color:#594adb ;
+  --icon-secondary-color: #f17c0e;
+    /* --icon-primary-color:#5468ff; darkblue*/
+      /* --icon-primary-color:#990000 ; 
+  --icon-secondary-color: #FFCC00; */ 
+
+  /* --icon-secondary-color : #7264e8; #f17c0e*/
```
Comment (haarsh24) [inline button/index.css]: Remove commented code if not necessary.

File: button/utility/component.css [hunk 4]
```diff
@@ -0,0 +1,255 @@
...
+
+        /* box-shadow: 5px 5px 20px 0 rgb(0 0 0 / 20%); */
+      }
+    
+    
+    
+      .component-footer-container{
+        color: #ffffff;
+        background-color: #333333;
+        text-align: center;
```
Comment (haarsh24) [inline button/utility/component.css]: Declare these colors and use.

File: button/utility/scrollbehaviour.css [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+  }
+  
+  ::-webkit-scrollbar-thumb {
+    background: var( --icon-primary-color);
+    border-radius: 10px;
+  }
+  
+  ::-webkit-scrollbar:horizontal{
+    height: 7px;
+    color: rgb(124, 136, 238);
```
Comment (haarsh24) [inline button/utility/scrollbehaviour.css]: Use same color codes.

**Review by haarsh24 — APPROVED** [review]
Some minor changes , Rest all looks good.
You can merge . Good work with UI,

**Review by haarsh24 — APPROVED** [review]

**Pull request merged**